<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGrade</class>
 <widget class="QWidget" name="AddGrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>233</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Grade</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(6, 12, 10);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_Modul">
          <property name="styleSheet">
           <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Modul</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Passed">
          <property name="styleSheet">
           <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Passed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Grade">
          <property name="styleSheet">
           <string notr="true">font: 700 11pt &quot;Graduate&quot; ;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Grade</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(6, 12, 10);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox_Modul">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(192, 191, 188);
color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Passed">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(192, 191, 188);
color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Grade">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(246, 245, 244);
color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
